docs/standard/library-guidance/strong-naming.md

```diff
@@ -46,6 +46,8 @@ You should strong name your open-source .NET libraries. Strong naming an assembl
 **✔️ CONSIDER** checking in the key used to strong name into your source control system.
```

Comment:
nit unrelated to your PR: checking in...into is redundant (should be "checking the key...into...")

Reply:
I changed it around a little. I think "check in" in the age of git is anachronistic.